data/utils/README.md

```diff
@@ -4,16 +4,23 @@ The `get-kaggle-dataset-meta-data.py` python script fetches the metadata for the
 
 ## Pre-requisites
 
-Setup your `.bashrc` or `.zshrc` or Windows environment with the below environment variables:
+- Python 3.10 or higher
+- Docker (to run inside docker containers)
+- Shell-scripting (basic skills)
+- Kaggle
+
+**Steps**
+
+- Setup your `.bashrc` or `.zshrc` or Windows environment with the below environment variables:
 
 ```bash
 export KAGGLE_USERNAME="[your kaggle username]"
 export KAGGLE_KEY="[your kaggle API key]"
 ```
 
-See [How to Obtain a Kaggle API Key](https://christianjmills.com/posts/kaggle-obtain-api-key-tutorial/).
+- See [How to Obtain a Kaggle API Key](https://christianjmills.com/posts/kaggle-obtain-api-key-tutorial/).
```

Comment:
I think it would be better to point towards kaggle's official docs at https://www.kaggle.com/docs/api since they are more likely to keep that up to date with any changes etc

Reply:
Apologies for the delay will do it asap

Reply:
Although the Specific topic "How to Obtain a Kaggle API Key" is better explained by Christian Mills than on Kaggle's docs - let me search for an equivalent one

Reply:
See [a9be773](https://github.com/neomatrix369/learning-path-index/pull/11/commits/a9be773179bba36fc2ecbd0bb4f2d210f0879ac2) - enhanced the links section a bit